app/models/settings/general.rb

```diff
@@ -2,8 +2,6 @@ module Settings
   class General < Base
     self.table_name = "site_configs"
 
-    HEX_COLOR_REGEX = /\A#(\h{6}|\h{3})\z/
```

Comment:
This constant is no longer used in this file, it was a leftover from the settings refactoring.